### PR TITLE
Add build config for logo

### PR DESCRIPTION
**Pull request by ibolton336**
This PR addresses the issue https://bugzilla.redhat.com/show_bug.cgi?id=1881970

Now we can add a .env file with the contents 
```
BRAND_TYPE=RedHat
```
at build time to override the default Konveyor brand setting. This will allow us to display the redhat logo downstream.

Note: As of January 2020, ES2020 supports Promise.allSettled out of the box removing the need for the Q library. This library was causing some dependency clashes with the dotenv webpack plugin, so it is removed as a part of this PR.

### CODE REVIEW

File: config/webpack.dev.js [hunk 1]
```diff
@@ -31,10 +31,11 @@ const PORT = process.env.PORT || localConfig.devServerPort;
 const EXPRESS_PORT = process.env.EXPRESS_PORT || 9001;
 
 const plugins = [
-  new webpack.NoEmitOnErrorsPlugin(),
```
Comment (mturley) [inline config/webpack.dev.js:34]: What's the reason for removing these other plugins?

Comment (ibolton336) [reply config/webpack.dev.js:34]: VScode had a line crossed through them saying they were deprecated. Cant find anything to support that. I will add them back in.

Comment (ibolton336) [reply config/webpack.dev.js:34]: image

Comment (mturley) [reply config/webpack.dev.js:34]: oh interesting ok. 👍

**Review by mturley — APPROVED** [review]
LGTM 👍

**Comment by eriknelson** [comment]
Trying to confirm this actually works building an image and deploying, will ack and merge as soon as I can do that

**Review by eriknelson — APPROVED** [review]
Confirmed the fix (using the correct env var helps..), LGTM and I've communicated the fix with the downstream guys, thank you!

**Pull request merged**